xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/serverinfo/ServerInfo.java

```diff
@@ -0,0 +1,281 @@
+    } else if (ClioServerInfo.class.isAssignableFrom(this.getClass())) {
+      return clioServerInfoMapper.apply((ClioServerInfo) this);
+    } else if (ReportingModeServerInfo.class.isAssignableFrom(this.getClass())) {
+      return reportingModeServerInfoMapper.apply((ReportingModeServerInfo) this);
+    } else {
+      throw new IllegalStateException(String.format("Unsupported ServerInfo Type: %s", this.getClass()));
+    }
+  }
+
+  /**
+   * Information on the most recently closed ledger that has not been validated by consensus. If the most recently
+   * validated ledger is available, the response omits this field and includes {@link #validatedLedger()} instead.  Per
+   * xrpl.org docs, this field is optionally present in any server response and may be omitted.
+   *
+   * @return An optionally-present {@link ServerInfoLedger} containing information about the server's view of the most
+   *   recently closed ledger.
+   */
+  @JsonProperty("closed_ledger")
+  Optional<ServerInfoLedger> closedLedger();
+
+  /**
+   * Range expression indicating the sequence numbers of the ledger versions the local rippled has in its database. This
+   * may be a disjoint sequence such as {@code 24900901-24900984,24901116-24901158}. If the server does not have any
+   * complete ledgers (for example, it recently started syncing with the network), this will be an empty
+   * {@link String}.
+   *
+   * @return A {@link String} representing a range of ledger sequences.
+   */
+  @JsonProperty("complete_ledgers")
+  @JsonDeserialize(using = CompleteLedgersDeserializer.class)
+  List<Range<UnsignedLong>> completeLedgers();
+
+  /**
+   * Determines if the supplied {@code ledgerIndex} exists on the rippled server by inspecting
+   * {@link #completeLedgers()}.
+   *
+   * @param ledgerIndex An {@link UnsignedLong} representing a particular ledger index.
+   *
+   * @return {@code true} if the rippled server includes {@code ledgerIndex} in its local database; {@code false}
+   *   otherwise.
+   */
+  @Value.Derived
+  @JsonIgnore
+  default boolean isLedgerInCompleteLedgers(final UnsignedLong ledgerIndex) {
+    return this.completeLedgers().stream()
+      .anyMatch(range -> range.contains(ledgerIndex));
+  }
+
+  /**
+   * The number of times (since starting up) that this server has had over 250 transactions waiting to be processed at
+   * once. A large number here may mean that your server is unable to handle the transaction load of the XRP Ledger
+   * network. For detailed recommendations of future-proof server specifications, see
+   * <a href="https://xrpl.org/capacity-planning.html">Capacity Planning</a>.
+   *
+   * @return An optionally-present {@link String}.
+   */
+  @JsonProperty("jq_trans_overflow")
+  Optional<String> jqTransOverflow();
+
+  /**
+   * (Admin only) Detailed information about the current load state of the server.
+   *
+   * @return An optionally-present {@link ServerInfoLoad}.
+   */
+  @JsonProperty("load")
```

Comment:
not present in clio

Reply:
Resolved via https://github.com/XRPLF/xrpl4j/pull/264/commits/4406297d16ef3348a81b1339e06a9842e8c2fb45